docs/docs/dcap.md

```diff
@@ -0,0 +1,111 @@
+## The DCAP protocol
+
+There are two protocols for establishing what code runs in an enclave: EPID and DCAP. EPID is an older
+protocol designed for consumer applications and includes some sophisticated privacy features. For servers
+where the IP address doesn't need to be hidden (because it's public in DNS, to begin with), these features
+aren't helpful, and thus there is DCAP (_Data Center Attestation Primitives_). DCAP requires more modern hardware
+but is otherwise simpler and more robust. You may also see DCAP referred to as "ECDSA attestation".
+
+!!! important
+Intel does not provide EPID attestation support on any Xeon Scalable processors including the Ice Lake Xeon CPUs and any future
```

Comment:
This is not rendered correctly

Reply:
Done. Also corrected other call-out boxes.